Setup/UpgradeSchema.php

```diff
@@ -385,6 +385,143 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
 
         // TODO: Add foreign key on avatax_cross_border_class.cross_border_type
```

Comment:
Remove this TODO, assuming the foreign key you added below is the one referenced here.

Reply:
It is not. I didn't add a foreign key, i added a unique index